src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -290,6 +292,26 @@ protected void _terminate(TaskListener listener) throws IOException, Interrupted
         listener.getLogger().println(msg);
     }
 
+    private void deleteSlavePvc(TaskListener listener, KubernetesClient client) throws IOException {
+        try {
+            for (String pvc_name : this.template.getPVCName(this)) {
+                Boolean deleted = client.persistentVolumeClaims().inNamespace(getNamespace()).withName(pvc_name).delete();
+                if (!Boolean.TRUE.equals(deleted)) {
+                    String msg = String.format("Failed to delete pvc %s/%s: not found", getNamespace(), pvc_name);
+                    LOGGER.log(Level.WARNING, msg);
+                    listener.error(msg);
+                    return;
```

Comment:
```suggestion
                    continue;
```
Otherwise you will leak pvcs.